### PR TITLE
Add dynamic rotation assignment support

### PR DESCRIPTION
**Pull request by loicgasser**
TODO:
- [x] Test Link
- [x] Document

For wind direction support.

Must be defined in radian clockwise.

This PR also  contains a fix for the generation of triangles. (fix in ol)

Their exemple still needs a fix https://openlayers.org/en/latest/examples/regularshape.html

### CODE REVIEW

File: src/components/StylesFromLiteralsService.js [hunk 1]
```diff
@@ -120,7 +119,8 @@ goog.provide('ga_stylesfromliterals_service');
           minResolution: getMinResolution(value),
           maxResolution: getMaxResolution(value),
           labelProperty: getLabelProperty(value.vectorOptions.label),
-          labelTemplate: getLabelTemplate(value.vectorOptions.label)
+          labelTemplate: getLabelTemplate(value.vectorOptions.label),
+          imageRotation: value.rotation
```
Comment (oterral) [inline src/components/StylesFromLiteralsService.js]: `imageRotationProperty` is more understandable.

Comment (loicgasser) [reply src/components/StylesFromLiteralsService.js]: I agree.

**Comment by loicgasser** [comment]
[Test Link](https://mf-chsdi3.dev.bgdi.ch/meteo_temp/shorten/7815fdb2a5)

**Comment by loicgasser** [comment]
comments addressed and I added a testlink ping @davidoesch @oterral

**Comment by loicgasser** [comment]
I rotation on station_symbol property which either 1 either 2.

**Pull request merged**